fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -1,44 +1,98 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
+use std::hint::black_box;
 
-use quick_xml::events::Event;
-use quick_xml::reader::Reader;
+use quick_xml::{events::Event, reader::Reader, writer::Writer};
 use std::io::Cursor;
 
-fuzz_target!(|data: &[u8]| {
-    // fuzzed code goes here
-    let cursor = Cursor::new(data);
-    let mut reader = Reader::from_reader(cursor);
+macro_rules! debug_and_pretty_print {
+    ($x:expr) => {
+        let _unused = std::hint::black_box(format!("{:?} {:#?}", $x, $x));
```

Comment:
I'm not against it, but I'm not sure if this will actually increase test coverage, as pretty printing is handled withing the `Formatter` object in uses where it is derived automatically, and in the case of our manual implementations, they are formatted as tuple-structs with no alternate pretty formatting.

Reply:
I don't think the macro should have `print` in the name if it's not printing anything though.  Maybe replace with `*format`.